src/main/java/org/jabref/logic/util/io/RegExpBasedFileFinder.java

```diff
@@ -57,58 +56,14 @@ public static String expandBrackets(String bracketString, BibEntry entry, BibDat
         Matcher m = SQUARE_BRACKETS_PATTERN.matcher(bracketString);
         StringBuffer s = new StringBuffer();
         while (m.find()) {
-            String replacement = getFieldAndFormat(m.group(), entry, database, keywordDelimiter);
+            String replacement = BracketedExpressionExpander.expandBrackets(m.group(), keywordDelimiter, entry, database);
```

Comment:
Since you are modifying this piece of code anyway, could you improve it a little and give `m` a more meaningful name, e.g. `matcher`.

Reply:
Fixed: m -> matcher, s -> expandedStringBuffer